frontend/simple/index.html

```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<html>
+<html lang="eng">
```

Comment:
Why is this being added? What is its purpose? Can it interfere with the i18n stuff? If it's not needed then please don't add this just in case it breaks i18n...

Reply:
When someone uses a screen reader, it will read with the device's default language (in my case Portuguese), unless the context says something different.
In this case, while testing Group Income with a screen reader, the english text was being read with Portuguese accent/pronounce and that's not what we want. So we need to say the site is eng in fact (and change it dynamically when the i18n changes). I can open an issue for that too as well :)